doc/modules/cross_validation.rst

```diff
@@ -947,49 +947,49 @@ Permutation test score
 ======================
 
 :func:`~sklearn.model_selection.permutation_test_score` offers another way
-to evaluate the performance of classifiers. It provides a permutation-based
+to evaluate the performance of estimators. It provides a permutation-based
```

Comment:
Actually, I believe it would work on anything that we can somehow make to score.